Index: platforms/android/src/com/plugin/gcm/GCMIntentService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- platforms/android/src/com/plugin/gcm/GCMIntentService.java	(revision )
+++ platforms/android/src/com/plugin/gcm/GCMIntentService.java	(revision )
@@ -70,7 +70,7 @@
 			PushPlugin.sendExtras(extras);
 
 			// Send a notification if there is a message
-			if (extras.getString("message").length() != 0) {
+			if (extras.getString("alert").length() != 0) {
 				createNotification(context, extras);
 			}
 		}
@@ -96,7 +96,7 @@
 				.setTicker(appName)
 				.setContentIntent(contentIntent);
 
-		String message = extras.getString("message");
+		String message = extras.getString("alert");
 		if (message != null) {
 			mBuilder.setContentText(message);
 		} else {
